src/mapper_class.cpp

```diff
@@ -323,6 +356,58 @@ void MapperClass::PublishRadiusMarkers(const Eigen::Vector3d &center,
     obstacle_radius_marker_pub_.publish(obstacle_radius_marker);
 }
 
+void MapperClass::PublishPathPlanningConfigMarkers(const pensa_msgs::PathPlanningConfig &path_planning_config,
+                                                   const std::string &inertial_frame_id) {
+    const std::string ns = "path_planning_config";
+    const double thickness = 0.1;
+    std_msgs::ColorRGBA no_fly_zone_color = visualization_functions::Color::Red();
+    std_msgs::ColorRGBA fly_zone_color = visualization_functions::Color::Green();
+    no_fly_zone_color.a = 0.3;  // make them slightly transparent
+    fly_zone_color.a = 0.15;    // make them slightly transparent
+    visualization_msgs::MarkerArray no_fly_zones_markers;
+    visualization_functions::DrawPathPlanningConfig(path_planning_config, ns, inertial_frame_id, no_fly_zone_color,
+                                                    fly_zone_color, thickness, &no_fly_zones_markers);
+    path_planning_config_pub_.publish(no_fly_zones_markers);
+}
+
+void MapperClass::PublishPathPlanningPathMarkers(const std::vector<Eigen::Vector3d> &path,
+                                                 const std::vector<Eigen::Vector3d> &pruned_path,
+                                                 const std::string &inertial_frame_id) {
+    visualization_msgs::MarkerArray markers;
+    const std_msgs::ColorRGBA color_path = visualization_functions::Color::Yellow();
+    const std_msgs::ColorRGBA color_pruned_path = visualization_functions::Color::Purple();
+    const std::string ns_path = "Astar";
+    const std::string ns_path_pruned = "Astar_pruned";
+    visualization_functions::PathVisualization(path, color_path, inertial_frame_id, ns_path, &markers);
+    visualization_functions::PathVisualization(pruned_path, color_pruned_path, inertial_frame_id,
+                                               ns_path_pruned, &markers);
+    path_planning_path_marker_pub_.publish(markers);
+}
+
+void MapperClass::LoadPathPlanningConfig(const std::string &inertial_frame_id,
```

Comment:
This function calls the service in `store_arbiter` to load the path planning configs